### ISSUE TITLE
[1.12.2] Thermal Foundation Resonant ender bucket to enderpearl recipe broken

### ISSUE THREAD

**Issue by Zedrimar** [CoFH/Feedback#637]
When trying to make enderpearls with cryothium dust and a resonant ender bucket, I get back 10 redstone rather than the 4 ender pearls

![image](https://user-images.githubusercontent.com/8153290/32872556-38e7dd1c-ca3c-11e7-80ac-6e7629ea7846.png)

![image](https://user-images.githubusercontent.com/8153290/32872573-4673e7e6-ca3c-11e7-9c35-fca186a61578.png)

Forge: *.2542
Thermal Foundation: *2.3.6.16

**Comment by KingLemming** [comment]
This doesn't seem to actually be a thing, no. Just tested it.

There's been some recipe issues with Forge lately. I suspect this might be one of them.

**Comment by KingLemming** [comment]
Will leave it open for discussion, but I can't make this happen and I'm quite sure it's not our fault.

**Comment by Zedrimar** [comment]
Unless my MC is somehow going screwy, I tested it with just the thermal expansion mods/modules along with JEI and I still got the bug :/

![image](https://user-images.githubusercontent.com/8153290/32906222-17766ec6-cab1-11e7-84da-0e525d576df0.png)

And the last mod: 

![image](https://user-images.githubusercontent.com/8153290/32906250-2da034e8-cab1-11e7-9c1f-49fb45693fa1.png)

**Comment by Zedrimar** [comment]
I literally got rid of ANY possible mod that could even have a remote chance of redirecting oredicts and somehow the mod still hates me XD

**Comment by Zedrimar** [comment]
did one last final test with just this: 

![image](https://user-images.githubusercontent.com/8153290/32906683-8a83372c-cab2-11e7-930f-d1fb668ccbce.png)

Still had the bug.... :/

**Comment by Zedrimar** [comment]
Upon further testing, I have concluded that you are indeed right, it is not on your end but instead on forge's. 

I tested with *.2515 version of forge and the bug disappeared. Going back to *.2544 caused the bug to appear so somewhere in between, forge screwed something up. Sadly my lack of modding experience means that I have no idea what to report to them.... Sorry for the trouble. I have to stay with latest version unfortunately because every time that BoP updates, they use the latest version and going back would crash the game :/
